src/groundlight/client.py

```diff
@@ -67,10 +67,18 @@ def __init__(self, endpoint: Optional[str] = None, api_token: Optional[str] = No
 
         configuration.api_key["ApiToken"] = api_token
 
+        self.api_token = api_token[:20] + "..."   # redact the secret
```

Comment:
Why `:20`? Is that partially revealing the secret? Or is the beginning of the key just boilerplate?

Reply:
API tokens consist of a unique KSUID and a random secret.  The `:20` truncates to just the KSUID, which is what we store in the backend - so it's the recoverable / identifiable part of the token.